### ISSUE TITLE
how many epochs and iterations are used for Lits dataset? 

### ISSUE THREAD

**Issue by sindhura234** [jackyko1991/vnet-tensorflow#22]
Thank you for sharing the code
I am working on some similar images like Lits

How many epoches and maximum iterations are being used to produce the results? 
Also optimizer and it learning rate?
